django/views/debug.py

```diff
@@ -294,7 +298,7 @@ def get_traceback_data(self):
             'settings': get_safe_settings(),
             'sys_executable': sys.executable,
             'sys_version_info': '%d.%d.%d' % sys.version_info[0:3],
-            'server_time': timezone.now(),
+            'server_time': localtime(datetime.utcnow().replace(tzinfo=utc)),
```

Comment:
I meant -- isn't the call to `localtime` going to raise the same exception as the template engine? Because the template engine calls this function or an equivalent one.